chart/sns-connector/README.md

```diff
@@ -0,0 +1,202 @@
+# OpenFaaS Pro SQS Connector
+
+The SNS connector can be used to invoke functions from an AWS SNS messages.
+
+## Prerequisites
+
+- Purchase a license
+
+  You will need an OpenFaaS License
+
+  Contact us to find out more [openfaas.com/pricing](https://www.openfaas.com/pricing)
+
+- Install OpenFaaS
+
+  You must have a working OpenFaaS installed.
+
+## Configure your secrets
+
+- Create the required secret with your OpenFaaS Pro license code:
+
+```bash
+$ kubectl create secret generic \
+    -n openfaas \
+    openfaas-license \
+    --from-file license=$HOME/.openfaas/LICENSE
+```
+
+- Create an AWS credentials secret:
```

Comment:
We should tell them what permissions this requires.

Reply:
@welteki ?

Reply:
Not sure what you mean by this. Do you mean the permission it needs to have within AWS?

Reply:
Yes.

Probably ConfirmSubscription and Subscribe - there are specific names on AWS